packages/ui/src/lib/hooks/useKeyboardShortcuts.ts

```diff
@@ -11,7 +11,6 @@ const SKIP_KBDS = [
 	'copy',
 	'cut',
 	'paste',
-	'delete',
```

Comment:
This was a sneaky manual skip for the "delete" action; I've removed that and our e2e tests are working.

Reply:
Ah, was a bit confused by the e2e tests and probably wouldn't have thought about this until much later 😅